lbry/testcase.py

```diff
@@ -220,6 +220,9 @@ def __init__(self, *args, **kwargs):
         self.account: Optional[Account] = None
 
     async def asyncSetUp(self):
+        if sys.version_info < (3, 8):
+            # hide warning about TaskWakeupMethWrapper, see bugs.python.org/issue38608
+            asyncio.get_running_loop().set_debug(False)
```

Comment:
This is outside the scope of the PR but also we don't want to turn off debug made. In fact, we explicitly turn it on in the same file so it doesn't make sense to then explicitly turn it off: https://github.com/lbryio/lbry-sdk/blob/75a1cc0d3305743b2a8a233d1169d31f5954cace/lbry/testcase.py#L125

Reply:
Fixed